docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+
+*When necessary, there will be a link to the ***advanced search section*** to change any other text as well*
+
+
+<details>
+<summary>**About Tooltips**</summary>
+:::info tooltips
+*To assist you*, I’ve added <Tooltip
+  text={<strong>tooltips</strong>}
+  tooltip="This is a tooltip"
+/>
+ throughout the tutorial to explain any tricky terms as you encounter them. 
+
+<Tooltip text={<><strong>Key terms</strong><span className="icon-alert"></span></>} tooltip="These will need the most attention" /> are marked with an exclamation icon when they're necessary. <Tooltip text={<italics>Hover over almost any</italics>} tooltip="almost" /> **bold** term for a quick explanation. Highlighted terms become easier to understand over the course of the tutorial.
+
+    >Explore the tooltips as needed for clarification
+:::
+</details>
+
+You will need: 
+-  <Tooltip text={<strong>GlacierKit</strong>} tooltip="A modding tool used for editing and managing game files in Hitman: World of Assassination." />
+- <Tooltip text={<strong>manifest file</strong>} tooltip="A file in your mod folder that holds important information and can execute code" />  
+- (Optionally) install <Tooltip text={<strong>VS code</strong>} tooltip="VS Code makes editing files smoother with features like helpful suggestions and quick fixes, making it easier to catch errors and stay organized as your mod grows." /> and use a <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> in your manifest.
+
+    ```json
+     "$schema": "https://raw.githubusercontent.com/atampy25/simple-mod-framework/main/Mod%20Manager/src/lib/manifest-schema.json"
+     ```
+---
+*If you're familiar with GlacierKit or feel confident enough, the [Quick Reference Guide](#quickreference) will make this tutorial turn into 2 minute shortcut.*
+
+---
+
+Start by opening your project/mod folder in  <Tooltip text={<strong>GlacierKit</strong>} tooltip="A modding tool used for editing and managing game files in Hitman: World of Assassination." />. 
+
+---
+
+
+## Pt. 1: **LOCATE YOUR LOCR** 
+*A ***LOCR File*** is basically a "container" for the game's text you see in game.* 
+
+
+### 1. Identify your LOCR
+ Below are some of their <Tooltip text={<><strong>HASH</strong><span className="icon-alert"></span></>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> lines: 
+- Season 1 suits: `00985A1100E5EDDC`,
+- Season 2 suits: `009F430D046716BE` ,
+- Season 3 suits: `004B8C5124A49543` 
+
+
+<details>
+<summary>about **HASH**</summary>
+:::info HASH
+You can interpret <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> lines as a file's specific address or coordinates. They are the file's *unique identifiers* and can bring us straight to the file.
+>You will constantly have to use these throughout modding.
```

Comment:
This is an odd explanation, also HASH is not an acronym

Reply:
Some terminology like this is explained here: https://hitman-resources.netlify.app/documentation/terminology/hash

Reply:
I'll keep it brief and add a link